### DISCUSSION TITLE
Dual monitor setup with screen duplication

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#606] [Q&A] [answered]
Can I duplicate screen on the second monitor? I use kiosk mode from https://github.com/ltsp/ltsp/wiki/xfreerdp so I can't set the duplication from the desktop environment. And xfreerdp with /multimon parameter extends the screen instead of mirroring. I guess I need to configure xorg.conf?

**Comment by alkisg** [comment, marked as answer]
Sure, from the [man page](https://ltsp.org/man/ltsp.conf/) read the example that says "If some clients need a custom xorg.conf file"...

Then follow some online tutorial on how to create an appropriate xorg.conf, for example https://wiki.ubuntu.com/X/Config/Multihead or https://wiki.archlinux.org/title/xorg.

**Reply by Mogaba** [reply]
Thanks, I got it working! The xorg.conf is very simple:
```
Section "Monitor"
    Identifier  "HDMI-0"
    Option      "Position" "0 0"
EndSection

Section "Monitor"
    Identifier  "VGA-0"
    Option      "Position" "0 0"
EndSection
```
The identifier can be obtained by running the `xrandr` command. If you connect with ssh, you need to specify the DISPLAY and XAUTHORITY variables:
`XAUTHORITY=/run/home/rdp/.Xauthority DISPLAY=:0 xrandr`

**Comment by ITaluone** [comment]
Hi

I have a similar issue: I am trying to boot a [Zbox MA760BE](https://www.zotac.com/de/product/mini_pcs/ma760#spec) which has four DP monitor outputs. But if I connect two screens, only one is detected and - because of that - duplicated. I cannot find an issue here, because if I boot other ZBoxes with e.g. HDMI and DVI output, it works fine. 

What I am missing here (pretty sure that I am missing something)? 

Thanks :)

Btw, applying an xorg config makes it even worse: still doubling, but squashing two screens into one :)

Edit: `xrandr` also detects only one screen

**Comment by alkisg** [comment]
@ITaluone it doesn't sound like an LTSP issue, try with a live CD or a local installation.
Once you find the solution there, you can use the same solution in LTSP too.

**Comment by Sistem-Pack** [comment]
Hello!
How I can use solution for Wayland?
I use two monitors on LTSP-client. First monitor support touchscreen.
My problem in touchscreen on the first screen which extends to both screens.
I need edit config for screen, but will it work for xorg.conf?

**Comment by alkisg** [comment]
@Sistem-Pack first solve the problem in a local installation.
If you need help for that, ask in your distribution channels.

After you have solved it locally, then if you have problems applying the solution in LTSP, tell us the solution and we'll help you with the LTSP part.
